ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -295,6 +295,9 @@ aaa_authorization:
   serial_console: < true | false >
   commands:
     all_default: < group group_name | local | none > < group group_name | local | none >
+    privilege:
+      - level: < privilege level(s) 0-15 >
+        group: < group_name | tacacs+ | radius | local | none >
```

Comment:
please use the same style as all the other areas:
`default: < group group_name | local | none > < group group_name | local | none >`
`local` and `none` are not groups, so the current code would produce invalid EOS config if only specifying `group: local`

Reply:
@RyanM-Arista the request was not just to update the README, but to update the actual data model. The key should be `default`, and you should not print out the word `group` as part of the template.

Reply:
@RyanM-Arista Thank for the latest update. Please note that the key should be `default` instead of `group` and the heading in the documentation table should be `User Stores` (not my favorite wording, but that is what we have elsewhere).
Thanks